src/Query/index.ts

```diff
@@ -109,7 +109,8 @@ export class Query {
     const searchPropertyMapping = getPropertyMapping();
 
     // if we need to boost for matching slug on an exact rawQuery match
-    if (strippedMapping[this.rawQuery.trim()]) {
+    const boostedStrings = strippedMapping[this.rawQuery.trim()];
+    if (Array.isArray(boostedStrings) && typeof boostedStrings[0] === 'string') {
       parts.push({
         text: {
           path: 'strippedSlug',
```

Comment:
You can also continue to use `boostedStrings` a few lines down rather than trim again. 

Also, I'm not as familiar with the intricacies of this. Is it enough to check the first element's type because there's no other combination of types in an array this could be, or should we go the extra mile and check all the array's element's types. Not blocking at all, just me wondering.

Reply:
good catch! done.

i think its valid to check since we have the type defined as 
```
interface ResultMapping {
  [key: string]: string[];
}
```